recipe/meta.yaml

```diff
@@ -52,19 +52,23 @@ test:
     - boto3 >=1.2.4
     - packaging
     #- pytest-catchlog
-    - mock  # [py2k]
+    - pytz
+    - pip
   files:
     - test_data
   commands:
```

Comment:
@ViridianMelody Could you update (if needed) tests from `run_test.bat`, `run_test.py`, and `run_test.sh` from the **conda-forge** recipe?